src/PrestaShopBundle/Resources/views/Admin/Improve/International/Tax/Blocks/form.html.twig

```diff
@@ -0,0 +1,73 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
```

Comment:
Please add a `{% block form %}` at the top to allow template easy inheritance

Reply:
@matks do we really need to add whole content inside block? What does that change? In addition, module developers cannot use these blocks I think.

Reply:
@matks I fixed it as sarjon suggested, plus tax_options (form_rest was missing there as well, I  guess its nothing wrong to fix it in this PR, as the changes are really minor).
Addressed all other comments as well.